packages/profile-sync-controller/src/sdk/authentication-jwt-bearer/types.ts

```diff
@@ -0,0 +1,72 @@
+import type { Env } from '../env';
+
+export const enum AuthType {
+  /* sign in using a private key derived from your secret recovery phrase (SRP). 
+       Uses message signing snap to perform this operation */
+  SRP = 'SRP',
+
+  /* sign in with Ethereum */
+  SiWE = 'SiWE',
+}
+
+export type AuthConfig = {
```

Comment:
Future note - maybe we can add some documentation to the types in future.
I believe they get picked up when docs are generated from types.